Builder examples don't work

### Please complete the following tasks

- [X] I have searched the [discussions](https://github.com/clap-rs/clap/discussions)
- [X] I have searched the [open](https://github.com/clap-rs/clap/issues) and [rejected](https://github.com/clap-rs/clap/issues?q=is%3Aissue+label%3AS-wont-fix+is%3Aclosed) issues

### Rust Version

rustc 1.58.1 (db9d1b20b 2022-01-20)

### Clap Version

3.0.14

### Minimal reproducible code

```rust
use clap::{app_from_crate, arg, App, AppSettings};

fn main() {
    let matches = app_from_crate!()
        .global_setting(AppSettings::PropagateVersion)
        .global_setting(AppSettings::UseLongFormatForHelpSubcommand)
        .setting(AppSettings::SubcommandRequiredElseHelp)
        .subcommand(
            App::new("add")
                .about("Adds files to myapp")
                .arg(arg!([NAME])),
        )
        .get_matches();

    match matches.subcommand() {
        Some(("add", sub_matches)) => println!(
            "'myapp add' was used, name is: {:?}",
            sub_matches.value_of("NAME")
        ),
        _ => unreachable!(
            "Exhausted list of subcommands and SubcommandRequiredElseHelp prevents `None`"
        ),
    }
}
```


### Steps to reproduce the bug with the above code

`cargo run`

### Actual Behaviour

error[E0432]: unresolved import `clap::app_from_crate`

### Expected Behaviour

should work

### Additional Context

_No response_

### Debug Output

_No response_

--- a/examples/cargo-example-derive.rs
+++ b/examples/cargo-example-derive.rs
@@ -1,3 +1,5 @@
+// Note: this requires the `derive` feature
+
 use clap::Parser;
 
 #[derive(Parser)]

--- a/examples/cargo-example.rs
+++ b/examples/cargo-example.rs
@@ -1,3 +1,5 @@
+// Note: this requires the `cargo` feature
+
 fn main() {
     let app = clap::App::new("cargo")
         .bin_name("cargo")

--- a/examples/git.rs
+++ b/examples/git.rs
@@ -1,3 +1,5 @@
+// Note: this requires the `cargo` feature
+
 use std::path::PathBuf;
 
 use clap::{arg, App, AppSettings};

--- a/examples/multicall-busybox.rs
+++ b/examples/multicall-busybox.rs
@@ -1,3 +1,5 @@
+// Note: this requires the `unstable-multicall` feature
+
 use std::process::exit;
 
 use clap::{App, AppSettings, Arg};

--- a/examples/multicall-hostname.rs
+++ b/examples/multicall-hostname.rs
@@ -1,3 +1,5 @@
+// Note: this requires the `unstable-multicall` feature
+
 use clap::{App, AppSettings};
 
 fn main() {

